modules/dxp/apps/commerce/commerce-shop-by-diagram-web/src/main/java/com/liferay/commerce/shop/by/diagram/admin/web/internal/util/CSDiagramSettingUtil.java

```diff
@@ -25,9 +25,16 @@
 
 /**
  * @author Andrea Sbarra
+ * @author Crescenzo Rega
  */
 public class CSDiagramSettingUtil {
 
+	public static FileEntry fetchFileEntry(CSDiagramSetting csDiagramSetting)
```

Comment:
Hey @CrescenzoRegaLF why did you do this? I think it's better to add the getFileVersion in this Util class rather then make this public wrapper of the same private method

Reply:
Yes, it could also be done as you suggest